internal/lm/resource_test.go

```diff
@@ -216,6 +216,36 @@ func TestGPUResourceLabeler(t *testing.T) {
 
 }
 
+func TestSanitise(t *testing.T) {
+	testCases := []struct {
+		input    string
+		expected string
+	}{
+		{
+			input:    "a space separated string",
+			expected: "a-space-separated-string",
+		},
+		{
+			input:    "some(thing)else",
+			expected: "somethingelse",
+		},
+		{
+			input:    "some ( thing )else",
+			expected: "some-thing-else",
+		},
+		{
+			input:    "NVIDIA-TITAN-X-(Pascal)",
+			expected: "NVIDIA-TITAN-X-Pascal",
+		},
```

Comment:
Not a blocker, but adding the following explicitly may be good.
```suggestion
		},
		{
				input: " input  with multiple   spaces   ",
				expected: "input-with-multiple-spaces",
		},
```

Reply:
Done